# simple message for object weight

# weight
bool hand_stopped
float32 weight
float32 mass
float32 density


float32 mass_2
float32 cx
float32 cy
float32 cz

float32 gx
float32 gy
float32 gz


# time estimation
time time_now


# end of estimation
bool cog_stop